src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataAjaxController.java

```diff
@@ -64,33 +65,31 @@ public ProjectSampleMetadataAjaxController(UIMetadataImportService metadataImpor
 	@PostMapping("/upload/setSampleColumn")
```

Comment:
Since you are not creating anything this should be a `@PutMapping`

Reply:
Again, that was an oversight. Please see 7a6200970002b1b687b352cbdad63a48ef3d7e5c.